src/main/Makefile.am

```diff
@@ -28,10 +28,11 @@ pcvc4_SOURCES = \
 	driver_unified.cpp
 pcvc4_LDADD = \
 	libmain.a \
-	@builddir@/../parser/libcvc4parser.la \
-	@builddir@/../libcvc4.la \
 	$(READLINE_LIBS) \
-	@builddir@/../lib/libreplacements.la
+	@builddir@/../lib/libreplacements.la \
+	@builddir@/../libcvc4.la \
+	@builddir@/../parser/libcvc4parser.la \
+	@builddir@/../util/libutil.la
```

Comment:
1. At the end of this log file: https://s3.amazonaws.com/archive.travis-ci.org/jobs/224182598/log.txt , you can see the kinds of errors. It is a bit weird that building worked on the cluster machine but not Travis.

2. Uh, didn't know that, thanks! That's good to know.